vit/config/config.sample.ini

```diff
@@ -221,6 +226,8 @@
 #     <Equals>
 #     <Space>
 #     <Semicolon>
+#     <Enter>
+#     <#>
```

Comment:
This may need to be added to the sample config, but doesn't seem to fit with this PR, can it be extracted/explained in a separate PR? I like keeping the commits as clean and focused as possible.

Reply:
Sure thing, I've got a few small documentation fixes in several different places actually, should I just make one pull request with all of them?

Reply:
Yep, let's do it that way, thanks!